indra/statements.py

```diff
@@ -506,15 +507,9 @@ def refinement_of(self, other, entity_hierarchy, mod_hierarchy):
         for other_mod_ix in range(len(other.mod)):
             mod_found = False
             other_mod = other.mod[other_mod_ix]
-            other_mod_pos = other.mod_pos[other_mod_ix]
             for self_mod_ix in range(len(self.mod)):
```

Comment:
Fixes in statements.py in https://github.com/sorgerlab/indra/pull/48/commits/68a7babb258755a90b6bd03bc339fd8f509e7264